services/rfq/relayer/pricer/fee_pricer.go

```
@@ -256,6 +262,76 @@
+		{Type: uint256Type}, // deadline
+		{Type: uint256Type}, // nonce
+		{Type: addressType}, // exclusivityRelayer
+		{Type: uint256Type}, // exclusivityEndTime
+		{Type: uint256Type}, // zapNative
+		{Type: bytesType},   // zapData
+	}
+
+	tx := quoteRequest.Transaction
+
+	// Pack the arguments
+	return args.Pack(
```

golangci-lint (1.62.2)

error returned from external package is unwrapped: sig: func (github.com/ethereum/go-ethereum/accounts/abi.Arguments).Pack(args ...interface{}) ([]byte, error)

(wrapcheck)